Spine-project/spine-engine
# Remote Spine Engine Server

PR for issue #16

To test:
- See installation and server start-up instructions in https://spine-toolbox.readthedocs.io/en/server/spine_engine_server.html

Related PR's:
- https://github.com/Spine-project/Spine-Toolbox/pull/1604
- https://github.com/Spine-project/spine-items/pull/62
